src/mac/main_mac.mm

```diff
@@ -49,11 +55,19 @@
       return env.Undefined();
     }
 
-    auto callback = info[0].As<Napi::Function>();
+    // Create the desktop notifications manager only if it's a supported macOS version
+    if (@available(macOS 10.14, *))
+    {
+      desktopNotificationsManager = [GHDesktopNotificationsManager new];
+    }
+    else
+    {
+      return env.Undefined();
+    }
```

Comment:
I can't do a `if (!@available(...)) { return }` because the compiler is not smart enough to understand everything outside the `if` is safe for `10.14` 🤦